src/main/scala/edu/ie3/simona/model/grid/GridModel.scala

```diff
@@ -82,6 +96,15 @@ case object GridModel {
       switches: Set[SwitchModel]
   )
 
+  /** Collection of grid-related control strategies
+    *
+    * @param transformerControlGroups
+    *   Transformer control groups
+    */
+  final case class GridControls(
+      transformerControlGroups: Set[ControlGroupModel]
+  )
+
```

Comment:
What do we need this class for when we could just hand over `transformerControlGroups: Set[ControlGroupModel]` (e.g. in GridModel constructor) as well?

Reply:
As discussed I thing this enables us later to include also other controls. But I'm open for discussion here.